rbd-target-api.py

```diff
@@ -1368,19 +1408,27 @@ def pre_reqs_errors():
     :return: list of configuration errors detected
     """
 
+    valid_dists = ["redhat"]
+    valid_versions = ['7.4']
+
     required_rpms = [
-        # {"name": "device-mapper-multipath",
-        #  "version": "0.4.9",
-        #  "release": "99.el7"},
         {"name": "python-rtslib",
-         "version": "2.1.fb57",
-         "release": "5.el7"}
+         "version": "2.1.fb64",
+         "release": "0.1.20170815.gitec364f3.el7cp"},
```

Comment:
The ```el7cp``` suffixes are downstream specific -- does the version checking still work if you just leave off the suffix?

Reply:
looks like it. as long as the installed version meets or exceeds the version given it should be fine. will update